src/components/Nav/index.tsx

```diff
@@ -1,20 +1,34 @@
 import React from 'react';
 import { Link } from 'react-router-dom';
+import './nav.css';
 
 const Nav: React.FC = () => {
-    return (
-        <ul>
-            <li>
-                <Link to="/">Home</Link>
-            </li>
-            <li>
-                <Link to="/about">About</Link>
-            </li>
-            <li>
-                <Link to="/crear-documento">Create Doc</Link>
-            </li>
+  return (
+    <header>
+      <nav>
+        <ul className="nav">
```

Comment:
Feels odd this list to be a single list, knowing that are separated elements. Is there any reason to join them, rather than make separated elements ?

Reply:
They are separated in "style" but they all correspond to main app navigation, that's the reason i put it all together by the same UL. Usually the logo is used to give the user the possibility of returning to the home page (i think this is more a UX thing).
Despite this, i can tell you that the nav does not need a list inside it, quoting w3.org: `A nav element doesn't have to contain a list, it can contain other kinds of content as well.`.
So i think it would be fine to just do (pseudo code) nav hx ul li li li ul nav. It's up to you.

Reply:
I would keep the logo inside the nav, but OUTSIDE the UL, in a separated element, so de UL is actually what is in the right side of the bar, and the logo is separated. That way you keep clean the inbetween of the elements, and also, don't need that nth-child.

In this case nth child is not that good just because is easily broken (just adding another element next to the logo)